Uno/NuGetPackageExplorer.Wasm/NuGetPackageExplorer.Wasm.csproj

```diff
@@ -75,6 +75,7 @@
     <PackageReference Include="Uno.Wasm.Bootstrap.DevServer" Version="8.0.0-dev.210" />
     <PackageReference Include="Microsoft.Windows.Compatibility" Version="6.0.6" />
     <PackageReference Include="Uno.Core" Version="4.0.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="7.0.12" Condition="$(IsMacOS)"/>
```

Comment:
This should not be needed, what's the issue when building on macos?

Reply:
@jeromelaban Without this package the build fails with this error
<img width="445" alt="image" src="https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/assets/20712372/bc0ee44f-39bd-44bc-9ba2-d5310181bfb7">

Reply:
maybe we can it as a `PrivateAsset`? I believe it's needed to build on macos but the project CI uses Windows as a build machine

Reply:
I see, is this a vsmac error?

Reply:
rider error... I didn't try vsmac because you know... (end of life)

Reply:
Got it. This would be a rider error more than a macos error. Blazor support is most likely hard coded there and does not know about uno's wasm debug host.